init.lua

```diff
@@ -350,30 +375,32 @@ local function lootItem(index, doWhat, button)
     local itemName = mq.TLO.Corpse.Item(index).Name()
     mq.cmdf('/nomodkey /shift /itemnotify loot%s %s', index, button)
     -- Looting of no drop items is currently disabled with no flag to enable anyways
-    -- added check to make sure the cursor isn't empty so we can exit the pause early.
-    mq.delay(5000, function() return mq.TLO.Window('ConfirmationDialogBox').Open() or not mq.TLO.Corpse.Item(index).NoDrop() or mq.TLO.Cursor() == nil end)
+    -- added check to make sure the cursor isn't empty so we can exit the pause early.-- or not mq.TLO.Corpse.Item(index).NoDrop() 
```

Comment:
is mq.TLO.Cursor going to be set to the looted item immediately by the time the `if doWhat == 'Destroy'` runs? iirc you needed at least like an mq.delay(1) after the looting command

Reply:
the cursor == nil check that the last PR added to this delay condition may have actually broken that if its still the case, since cursor probably was always nil at the start of the delay.